packages/content/src/components/PageRating/option.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { StyledInput, StyledLabel } from './styled';
+import { IconProps } from './types';
+
+const Option: React.FC<IconProps> = ({ isSelected, value, onChange, children }: IconProps) => {
+    return (
+        <StyledLabel isSelected={isSelected}>
+            <StyledInput
+                type="radio"
+                name="pageRating"
```

Comment:
we used to pass a name prop to use here and if it is not passed then we generate a one with our uniqueId helper, in this case you should use ```useUniqueIdentifier```